lib/stat_tracker.rb

```diff
@@ -158,9 +157,9 @@ def team_by_id #helper method for issue #14
     @teams.values_at(:team_id, :teamname).to_h
   end
 
-  def average_scores_by_team_id(game_type) #helper method for issue #14
+  def average_scores_by_team_id(*game_type) #helper method for issue #14
```

Comment:
You might want to  - add your issue number to the code comment for any helper method you use. I think it will help us when we refactor and split things up next Iteration to know what methods are using which helpers. (Hopefully, save future us a little bit of headache 😃 )

Reply:
Discussed on zoom. Will implement later.